### PR TITLE
DOC: add documentation about how to handle new array printing

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

**Comment by charris** [comment]
Wrong markup language, rst is not markdown. Hit the `view` button on the file to see what it looks like.

**Comment by charris** [comment]
Might be better to find someplace in the documentation to put this, maybe the user guide, but I can't find a good spot.

File: doc/release/1.14.0-notes.rst [hunk 1]
```diff
@@ -307,6 +307,25 @@ In summary, the major changes are:
 * User-defined ``dtypes`` (subclasses of ``np.generic``) now need to
   implement ``__str__`` and ``__repr__``.
 
+You may want to do something like:
+```
+# FIXME remove next 3 lines once 1.14 is minimum NumPy requirement
+# and fix printing doctests to use new format.
+from distutils.version import LooseVersion
+if LooseVersion(np.__version__) >  LooseVersion("1.13.3"):
```
Comment (eric-wieser) [inline doc/release/1.14.0-notes.rst]: This should be recommending `LooseVersion(np.__version__) >= LooseVersion("1.14.0")` instead.

Or maybe just

```
try:
    np.set_printoptions(legacy="1.13")
except TypeError:
    pass

Comment (jarrodmillman) [reply doc/release/1.14.0-notes.rst]: That is what I ended up using for networkx (https://github.com/networkx/networkx/pull/2810).  I modified the commit accordingly and switched to rst.  I am not sure where this should go (assuming it is desirable), so I will leave it here until someone has a better suggestion.

**Comment by charris** [comment]
Thanks @jarrodmillman . I'll reword this a bit before the final release, but that will be easier to do with this in.

**Comment by eric-wieser** [comment]
Just to remind ourselves that this is on master, not 1.14

**Comment by charris** [comment]
@eric-wieser Yep, I've already modified the notes in the 1.14.x branch and will forward port various things after the release.

**Comment by eric-wieser** [comment]
Right, so this is making things harder since you now need to both back and forward-port the changes

**Comment by charris** [comment]
Forward and back is easy, I just checked out the file from master into the existing prep branch, modified it, and committed. When it comes time to forward port, I will just check out the modified files from the maintenance branch and commit them.

**Pull request merged**